src/components/Comments.js

```diff
@@ -0,0 +1,253 @@
+import React from 'react';
```

Comment:
Add the flow annotation.
```suggestion
// @flow
import React from 'react';
```